modules/ROOT/pages/customize.adoc

```diff
@@ -44,6 +45,7 @@ Components that are typically used in *API details* pages include:
 *API Console*, which renders information from the API RAML specification, and *Feed*, which displays comments,
 discussions, and questions.
 * *API Access Requestor*: Registers a client application and requests access to the API.
+* *Learn More Button*: Enables users to access a custom page with additional information about the API.
```

Comment:
Provides additional information about the API.

Reply:
Fixed.